### ISSUE TITLE
Support .NET 9.0

### ISSUE THREAD

**Issue by mike8161990** [jburzynski/TypeGen#210]
We're in the middle of upgrading our projects and CI pipeline to use .NET 9.0, however we discovered the CLI tool dotnet-typegen seems to require the .NET 8.0 runtime installed to function properly. This makes it difficult for our CI to use it with a simple .NET 9.0 SDK docker image for our CI (e.g. `mcr.microsoft.com/dotnet/sdk:9.0.100-noble`) as it does not have the older runtimes installed by default.

Can this project please be updated to support .NET 9.0?

**Comment by mike8161990** [comment]
As a workaround, the tool seems to work fine when using the [roll forward option](https://learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-9/sdk#net-tool-roll-forward).

**Comment by jburzynski** [comment]
I just released version 6.0.0, which upgrades TypeGen to .NET 9.0 - let me know in case there are any issues.

**Comment by mike8161990** [comment]
That works, thank you!

---

One thing I did notice however is that generated files which have import statements seem to now have mixed line endings when running on a Linux machine.

Previously on 5.0.1, I'd see output like this:
```
/**\r\n
 * This is a TypeGen auto-generated file.\r\n
 * Any changes made to this file can be lost when this file is regenerated.\r\n
 */\r\n
\r\n
import { MinimalExample } from './minimal-example';\r\n
\r\n
export interface Demo {\r\n
  demo: MinimalExample;\r\n
}\r\n
```

But now on 6.0.0, that carriage return after the import went away but all the others stayed like this:
```
/**\r\n
 * This is a TypeGen auto-generated file.\r\n
 * Any changes made to this file can be lost when this file is regenerated.\r\n
 */\r\n
\r\n
import { MinimalExample } from './minimal-example';\r\n
\n
export interface Demo {\r\n
  demo: MinimalExample;\r\n
}\r\n
```

Files without imports stayed the same.

I did a local build to inspect if this had something to do with https://github.com/jburzynski/TypeGen/pull/208, but when I ran the CLI tool using my own build everything worked as intended (all carriage returns removed). Now I'm kind of wondering if this release had some build issue? Not sure what to make of that.

**Comment by jburzynski** [comment]
I reverted #208 and published the change in 6.0.1.

I think the issue is because templates have fixed line endings (`\r\n` if you build TypeGen in Windows and `\n` if you build in Linux/Mac/Unix), so if `Environment.NewLine` is used for imports, it can cause mixed line endings - so for now I just reverted the change from #208.

**Comment by jburzynski** [comment]
I just released 6.0.2 - I brought back the changes from #208, but also now line endings are normalized to `Environment.NewLine` in the generated files, so it should solve the issue.

**Comment by mike8161990** [comment]
Confirmed I'm seeing only `\n` line endings on all generated files now. Thank you for digging into this!
